### ISSUE TITLE
AoT compilation fails because of CookieOptions

### ISSUE THREAD

**Issue by DirkWillem** [salemdar/angular2-cookies#19]
When I try to compile an angular 2 application with CookieService using the AoT compiler, it doesn't compile because of the following error: 

```
..../app.module.ngfactory.ts(436,82): error TS2346: Supplied parameters do not match any signature of call target.
```

When I take a look at that line, it looks like this:

```
if ((this.__CookieService_54 == (null as any))) { (this.__CookieService_54 = new import36.CookieService(); }
```

According to IntelliSense, the CookieService takes one required argument named `defaultOptions`, but it isn't provided by the compiled code. In my NgModule, I provide CookieService like this:

``` typescript
@NgModule({
    imports: [
        BrowserModule,
        SharedModule,
        FormsModule,
        routing
    ],
    declarations: [
        AppComponent,
        ...frameworkComponents,
        ...viewComponents,
        LoginComponent],
    providers: [
        appRoutingProviders,
        CookieService
    ],
    bootstrap: [AppComponent]
})
```

**Comment by salemdar** [comment]
Thank you for the detailed report, I will inspect this as soon as possible.

**Comment by DirkWillem** [comment]
Solved by #20
